src/main/java/com/google/maps/DirectionsApiRequest.java

```diff
@@ -83,7 +83,7 @@ public DirectionsApiRequest destination(String destination) {
   }
 
   /**
-   * The origin, as a latitude,longitude location.
+   * The origin, as a latitude/longitude location.
```

Comment:
I browsed through the whole source tree, and "latitude/longitude" seems to be a lot more common that "latitude,longitude". Seems like it should have parens and a space if it's going to be "(latitude, longitude)".